File: python/examples/special_elements/special_elements.py
"""Application logic for special_elements.uimd - demonstrates InfoLabel animation."""

import sys
import os

sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))

from runtime import UIApplication
from special_elements_ui import SpecialElementsUI


def main():
    app = UIApplication()
    window = SpecialElementsUI()
    window._app = app
    app.open(window)
    return app.run()


if __name__ == "__main__":
    sys.exit(main())

File: java/examples/special_elements/SpecialElements.java
import uimd.GeneratedWindowRuntime;

public class SpecialElements extends SpecialElementsUI
{
    public static void main(String[] arguments)
    {
        int exitCode = GeneratedWindowRuntime.runGeneratedAppMain(() ->
        {
            SpecialElements application = new SpecialElements();
            return GeneratedWindowRuntime.runGeneratedWindow(
                application,
                application.runtimeOptions(),
                arguments);
        });
        System.exit(exitCode);
    }
}
